lib/features/reps_core/reps_core.install

```diff
@@ -389,3 +389,81 @@ function reps_core_update_7012() {
     $source->startImport();
   }
 }
+
+/**
+ * Forcing reerting reps_core to apply upgrade 7014.
+ */
+function reps_core_update_7013() {
```

Comment:
This can also be requested to QA via drush. You just need to give him the order of the commands to be executed (drush fr then drush updb). But anyway, it's just a hook_update, but still makes a bit more core in the codebase.

Reply:
By using the install-clone from toolkit the install was going wrong because hook_updates are run automatically after the setting of the database. Using this ensure that fields are correctly set before making value changes